### ISSUE TITLE
[FEATURE REQUEST] reduce image size. 

### ISSUE THREAD

**Issue by JJger148** [sissilab/obsidian-image-toolkit#25]
If it is possible to reduce the size of the Image, a lot of storage space would be saved. 
Also it would be amazing if we could cut the image.

**Comment by sissilab** [comment]
> If it is possible to reduce the size of the Image, a lot of storage space would be saved. Also it would be amazing if we could cut the image.

Hi, did you mean the **size** is **storage size**? For example, if an image is 3MB, you want to reduce it to lower size?
BTW, if it's what you want, the reduced image may reduce the quality or resolution of the image.

**Comment by JJger148** [comment]
Yes that's what I want very often. Reduce the resolution for, so that it takes less space, but also to have only a small image displayed. 
Often I include book cover in my vault and I want them to be displayed just so that I can recognize it visually

**Comment by DryIce1** [comment]
Yeah Image Resize would be useful! Drag the corner of the image, and it resizes (usually wanting to make an image that is much to large a smaller more useful size for a standard screen). The aspect ratio should remain the same.

Something like this:
![image](https://user-images.githubusercontent.com/59501376/142977614-eecd4028-c223-48d4-8d5b-50537afadcb3.png)
